rest-api-spec/src/main/resources/rest-api-spec/test/suggest/30_context.yml

```diff
@@ -195,7 +195,8 @@ setup:
       indices.refresh: {}
 
   - do:
-      indices.get_mapping: {}
+      indices.get_mapping:
+        include_type_name: false
```

Comment:
Is this necessary for mixed cluster?

Reply:
It is not technically necessary, since we don't use the output of the call. However, I just tried to use `include_type_name = false` for all mixed-cluster set-ups for clarity/ consistency, as opposed to making individual calls on a test-by-test basis.